### ISSUE TITLE
Q: custom normalization between samples?

### ISSUE THREAD

**Issue by tbrunetti** [macs3-project/MACS#510]
**Use case**
I have a series of ChIP-seq samples that I eventually want to perform differential peak calling and motif enrichment analysis upon between different treatment groups (and input).   

**Describe the problem**
I would like to use macs3 (or macs2) to call peaks but I want to apply and test a series of custom normalization methods to the samples instead of using macs size/normalization factor calculations. Am I able to bypass the normalization step in macs3 to perform all other steps besides the read normalization? 
 
If so, can you direct me to the location in the docs or some steps I would need to take in order to achieve this?

Thank you!

**Comment by taoliu** [comment]
@tbrunetti Yes you can. Please refer to this wikipage on how to do this: https://github.com/macs3-project/MACS/wiki/Advanced%3A-Call-peaks-using-MACS2-subcommands

Basically, you can get the pileup of treatment and control separately then apply your normalization on the bedGraph, optionally then combine them (if you want) by calculating the p-value based on MACS local poisson model, then call peaks directly on the bedGraph using `bdgpeakcall` command.

**Comment by tbrunetti** [comment]
This is very helpful, thank you!  Out of curiosity are these commands also supported on macs3, or should I strictly use macs2?
